### PR TITLE
doc: expermiental support for nrf53

### PR DESCRIPTION
**Pull request by ktaborowski**
[KRKNWK-17106]

### CODE REVIEW

File: doc/sidewalk_known_issues.rst [hunk 1]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+    Only on nRF5340 (experimental support). Failing certifiaction test.
+
+* KRKNWK-17321: Multiple restarts ends with fail.
+    Only on nRF5340 (experimental support). Failing certifiaction test.
+
+* KRKNWK-17308: Release version does not boot
+    Only on nRF5340 (experimental support).
+
+* KRKNWK-17114: Bootloder configuration not optimized.
+    Only on nRF5340 (experimental support). Large booloder partioin size 48 kB. DFU always requires external flash.
```
Comment (wojciechpluszynski) [inline doc/sidewalk_known_issues.rst:21]: spelling errors 
-> bootloader
->partition

File: doc/sidewalk_known_issues.rst [hunk 2]
```diff
@@ -5,10 +5,23 @@ Known issues
 
 See the list of known issues that are valid for the current state of development.
 
-v1.14.4
--------
+v1.14.5-dev1
+------------
+
+* KRKNWK-17320: BLE connection between Gateway and End Node is dropped earlier than 30 seconds of being idle.
+    Only on nRF5340 (experimental support). Failing certifiaction test.
```
Comment (wojciechpluszynski) [inline doc/sidewalk_known_issues.rst:12]: spelling error:
-> certification

File: doc/sidewalk_known_issues.rst [hunk 3]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+
+* KRKNWK-17321: Multiple restarts ends with fail.
+    Only on nRF5340 (experimental support). Failing certifiaction test.
```
Comment (wojciechpluszynski) [inline doc/sidewalk_known_issues.rst:15]: spelling error:
-> certification

File: doc/sidewalk_known_issues.rst [hunk 4]
```diff
@@ -5,10 +5,23 @@ Known issues
 
 See the list of known issues that are valid for the current state of development.
 
-v1.14.4
--------
+v1.14.5-dev1
```
Comment (wojciechpluszynski) [inline doc/sidewalk_known_issues.rst:8]: version of tag to be checked, before was discussed 1.14.4 dev1 as next release is going to be 1.14.5 dev1

**Review by wojciechpluszynski — CHANGES_REQUESTED** [review]

File: doc/sidewalk_known_issues.rst (hunk 4 above)
Comment (ktaborowski) [reply doc/sidewalk_known_issues.rst:8]: As discussed offline - v1.14.5 dev1

File: doc/sidewalk_known_issues.rst [hunk 5]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+------------
+
+* KRKNWK-17320: BLE connection between Gateway and End Node is dropped earlier than 30 seconds of being idle.
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:11]:
```suggestion
* KRKNWK-17320: Bluetooth LE connection between Gateway and End Node drops within 30 seconds of being idle.
```
tbc

File: doc/sidewalk_known_issues.rst [hunk 6]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+    Only on nRF5340 (experimental support). Failing certification test.
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:12]:
```suggestion
    The issue occurs only on the nRF5340 development kit (experimental support) and happens due to failing certification test.
```
rephrased it a bit - let me know if the meaning is affected

File: doc/sidewalk_known_issues.rst [hunk 7]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+
+* KRKNWK-17321: Multiple restarts ends with fail.
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:14]:
```suggestion
* KRKNWK-17321: The device fails after multiple restarts.
```
tbc - the device/application?

File: doc/sidewalk_known_issues.rst [hunk 8]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+    Only on nRF5340 (experimental support). Failing certification test.
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:15]:
```suggestion
   The issue occurs only on the nRF5340 development kit (experimental support) and happens due to failing certification test.
```
tbc - as above

File: doc/sidewalk_known_issues.rst [hunk 9]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+
+* KRKNWK-17308: Release version does not boot
+    Only on nRF5340 (experimental support).
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:18]:
```suggestion
    The issue occurs only on the nRF5340 development kit (experimental support).
```

File: doc/sidewalk_release_notes.rst [hunk 10]
```diff
@@ -5,6 +5,12 @@ Release notes
 
 See the release notes for the information about specific Sidewalk support releases for nRF Connect SDK.
 
+[v1.14.5-dev1] - 12/07/2023
+---------------------------
+
+* Added experimental support for nRF5340.
```
Comment (annwoj) [inline doc/sidewalk_release_notes.rst:11]:
```suggestion
* Added experimental support for the nRF5340 development kit.
```

File: doc/sidewalk_known_issues.rst [hunk 11]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+
+* KRKNWK-17114: Bootloder configuration not optimized.
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:20]:
```suggestion
* KRKNWK-17114: Bootloader configuration is not optimized
```

File: doc/sidewalk_known_issues.rst [hunk 12]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+    Only on nRF5340 (experimental support). Large bootloader partition size 48 kB. DFU always requires external flash.
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:21]:
```suggestion
   The issue occurs only on the nRF5340 development kit (experimental support). 
   Due to large bootloader partition size (48 kB), the DFU always requires an external flash.
```
TBC if the meaning is as intended

File: doc/sidewalk_known_issues.rst [hunk 13]
```diff
@@ -5,10 +5,23 @@ Known issues
...
+v1.14.5-dev1
+------------
+
+* KRKNWK-17320: BLE connection between Gateway and End Node is dropped earlier than 30 seconds of being idle.
+    Only on nRF5340 (experimental support). Failing certification test.
+
+* KRKNWK-17321: Multiple restarts ends with fail.
+    Only on nRF5340 (experimental support). Failing certification test.
+
+* KRKNWK-17308: Release version does not boot
```
Comment (annwoj) [inline doc/sidewalk_known_issues.rst:17]:
```suggestion
* KRKNWK-17308: Release version does not boot.
```

File: doc/sidewalk_release_notes.rst [hunk 14]
```diff
@@ -5,6 +5,12 @@ Release notes
...
+* Updateed sdk-nrf to v2.4.99 (main).
```
Comment (wojciechpluszynski) [inline doc/sidewalk_release_notes.rst:12]: Updated

File: doc/sidewalk_known_issues.rst (hunk 11 above)
Comment (wojciechpluszynski) [inline doc/sidewalk_known_issues.rst:20]: Bootloader

**Review by wojciechpluszynski — CHANGES_REQUESTED** [review]

**Review by annwoj — APPROVED** [review]

**Review by wojciechpluszynski — APPROVED** [review]
Approved

**Pull request merged**